pkg/agent/datamodel/defaults-scheduler.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
also don't need this

Reply:
PR to remove it is here: https://github.com/Azure/AgentBaker/pull/196